gamestonk_terminal/README.md

```diff
@@ -274,9 +274,14 @@ Command|Description
 
 Command|Description
 ------ | ------------
-`load`| load cryptocurrency data
-`view`| load and view cryptocurrency data
+`load`| load cryptocurrency data from coingecko
+`view`| view loaded cryptocurrency data
+`trend`| view top 7 coins from coingecko
 `top` | view top coins from coinmarketcap
+`add` | add binance ticker
```

Comment:
I don't really like the "add" keyword here since we are not adding to anything, but selecting it.

Instead, could we have:
```
`select` | select coin-currency pair
```